stock_by_warehouse/i18n/es.po

```diff
@@ -0,0 +1,173 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* stock_by_warehouse
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 16.0+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-03-20 18:29+0000\n"
+"PO-Revision-Date: 2023-03-20 18:29+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: stock_by_warehouse
+#. odoo-javascript
+#: code:addons/stock_by_warehouse/static/src/components/warehouse/warehouse_field.xml:0
+#, python-format
+msgid "... see all warehouses .."
+msgstr "... ver todos los almacenes .."
+
+#. module: stock_by_warehouse
+#. odoo-javascript
+#: code:addons/stock_by_warehouse/static/src/components/warehouse/warehouse_field.xml:0
+#, python-format
+msgid "Available now but not necessarily what I can sale"
+msgstr "Cantidad disponible pero no necesariamente puede ser vendida"
+
+#. module: stock_by_warehouse
+#. odoo-javascript
+#: code:addons/stock_by_warehouse/static/src/components/warehouse/warehouse_field.xml:0
+#, python-format
+msgid "I can sale this quantity now and deliver it immediately"
+msgstr "Puedo vender esta cantidad ahora y entregarla inmediatamente"
+
+#. module: stock_by_warehouse
+#. odoo-javascript
+#: code:addons/stock_by_warehouse/static/src/components/warehouse/warehouse_field.xml:0
+#, python-format
+msgid "Is coming in"
+msgstr "Esta llegando"
+
+#. module: stock_by_warehouse
+#. odoo-javascript
+#: code:addons/stock_by_warehouse/static/src/components/warehouse/warehouse_field.xml:0
+#, python-format
+msgid "Location"
+msgstr "Ubicación"
+
+#. module: stock_by_warehouse
+#. odoo-javascript
+#: code:addons/stock_by_warehouse/static/src/components/warehouse/warehouse_field.xml:0
+#, python-format
+msgid "Locations:"
+msgstr "Ubicaciones:"
+
+#. module: stock_by_warehouse
+#. odoo-javascript
+#: code:addons/stock_by_warehouse/static/src/components/warehouse/warehouse_field.xml:0
+#, python-format
+msgid "Outgoing"
+msgstr "Saliente"
+
+#. module: stock_by_warehouse
+#: model:ir.model,name:stock_by_warehouse.model_product_template
+msgid "Product"
+msgstr "Plantilla de producto"
```

Comment:
It doesn't match.

The same occurs in product variants

Reply:
I've double checked this and, in this case, what doesn't match is the original term. Check the `product` module

Reply:
Oh, I see, original term has changed, see https://github.com/odoo/odoo/commit/c2f394253ef329c62497b8d9986b36183354e474#diff-921ec733c3821301f62cd0d96e45990e7636f5db56a10ae76aed009e380bf149R18

 The one that it's outdated (this comes from Odoo) is the translation.

Let's fix the translation here and in Odoo.

Reply:
Hi, @luisg123v , ready and the link I think is https://www.transifex.com/odoo/odoo-16/translate/#es/product/437962391